lib/pbench/cli/agent/commands/tools/clear.py

```diff
@@ -87,6 +89,10 @@ def _clear_remotes(self, group: str) -> Tuple[int, Dict[str, list]]:
                     )
 
         for remote in remotes:
+            if not remote:
+                self.logger.warn("Remote host name can not be empty if specified")
```

Comment:
"can not be empty if specified" sounds awkward. Maybe just drop the "if specified". But since we're only warning about it, "can not" is rather strong, implying an error: it's more like "should not". Or, "we'd mildly prefer you don't specify empty remotes, or we'll have to nag you again".